editor/source/graphics/GridGraphic.js

```diff
@@ -106,6 +117,8 @@ export default class GridGraphic extends Graphic {
 
   /** Populate tiles based on given TopoJSON-backed map graphic */
   populateTiles(mapGraphic) {
+    if (this._lastTileEdge === settings.tileEdge) return
```

Comment:
Cool, I was also thinking that maybe we don't need to re-populate the tiles on every frame!

But I wonder if there's a more general approach—rather than do this specific check here, we might just remove the call to `updateTiles()` from `Canvas._render()` altogether—and just make sure that it _is_ called when any relevant property is updated. (In the near term, that may just be building another `tileEdgeSetting.onChange` handler, but could include other grid-changing operations in the future.)